lib/iknow_params/serializer.rb

```diff
@@ -95,8 +96,11 @@ def initialize
       super(::Integer)
     end
 
-    def load(str)
-      Integer(str)
+    # JSON only supports floats, so we have to accept a value
+    # which may have already been parsed into a Ruby Float or Integer.
```

Comment:
Rails seems to make this happen somehow. I'll see if I can find out where, but I was able to submit `4.0` and have it be a `Fixnum` `4`.